### DISCUSSION TITLE
can not run after upgrade from alpine 3.18 to alpine 3.19

### DISCUSSION THREAD

**Discussion by yyysuo** [wg-easy/wg-easy#976] [Q&A]
OS Information linux x86_64 Alpine Linux v3.19
Kernel Version 6.6.14-0-virt
Docker Version 24.0.7 (API: 1.43)

Container log

[#] ip -4 address add 10.8.0.1/24 dev wg0
[#] ip link set mtu 1420 up dev wg0
[#] iptables -t nat -A POSTROUTING -s 10.8.0.0/24 -o eth0 -j MASQUERADE; iptables -A INPUT -p udp -m udp --dport 51820 -j ACCEPT; iptables -A FORWARD -i wg0 -j ACCEPT; iptables -A FORWARD -o wg0 -j ACCEPT;
modprobe: can't change directory to '/lib/modules': No such file or directory
modprobe: can't change directory to '/lib/modules': No such file or directory
iptables v1.8.3 (legacy): can't initialize iptables table `nat': Table does not exist (do you need to insmod?)
Perhaps iptables or your kernel needs to be upgraded.
[#] ip link delete dev wg0
    at ChildProcess.exithandler (child_process.js:383:12)
    at ChildProcess.emit (events.js:400:28)
    at maybeClose (internal/child_process.js:1058:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:293:5) {
  killed: false,
  code: 3,
  signal: null,
  cmd: 'wg-quick up wg0'
}

**Comment by amitnpl** [comment]
I am running into same issue, any fix for this ?

**Comment by amitnpl** [comment]
for me , it looks like the image has iptables legacy although my host system is alpine 3.19, I can get it to work by manually removing iptables legacy and installing iptables nftables version, however changes is lost right after :(

**Comment by amitnpl** [comment]
I had to rebuild the image locally after commenting this line in the Dockerfile : 

```
#.Use.iptables-legacy
#RUN.update-alternatives.--install./sbin/iptables.iptables./sbin/iptables-legacy.10.--slave./sbin/iptables-restore.iptables-restore./sbin/iptables-legacy-restore.--slave./sbin/iptables-save.iptables-save./sbin/iptables-legacy-save
```

**Comment by pheiduck** [comment]
You have to modeprobe iptables

```
sudo modprobe ip_tables
sudo echo 'ip_tables' >> /etc/modules
```

**Reply by tryweb** [reply]
After executing this command on the docker host (Alpine 3.19) and restarting the docker host, I can see that the wg-easy container service is running normally. Thanks for providing this information.
